webrender_api/src/font.rs

```diff
@@ -93,6 +93,7 @@ pub enum FontRenderMode {
     Mono = 0,
     Alpha,
     Subpixel,
+    Bitmap,
```

Comment:
Would `Color` be a better name? When I read the word `Bitmap` I think of a monochrome pixel font. @jrmuizel felt the same when I asked him about it.